docs/setup_docs.sh

```diff
@@ -54,7 +54,7 @@ cd tmp
 # Since there's no documentation yet available for a release branch,
 # we only get the documentation from the main branch
 integrate_connector_docs elasticsearch v3.0.0
-integrate_connector_docs aws v3.0.0
+integrate_connector_docs aws v4.0
```

Comment:
Is this intentionally not using the v4.0.0 _tag_?
I don't mind pointing to the branch, just wanted to be sure.

Reply:
Yes, I am pointing to the branch since the tag had an error on the shortcode